Tests/HarmonyTests/DeleteNetworkDataSourceTests.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  DeleteNetworkDataSourceTests.swift
+//  HarmonyTests
+//
+//  Created by Borja Arias Drake on 11.10.2022..
+//
+
+import Alamofire
+import Foundation
+import Harmony
+import Nimble
+import XCTest
+
+@available(iOS 13.0, *)
+final class DeleteNetworkDataSourceTests: XCTestCase {
+
+    private typealias Utils = GenericDataSourceUtils
+    
+    func test_deleteAll_allobjects_query_not_supported() {
+        let dataSource: DeleteNetworkDataSource = provideDeleteDataSource(url: "")
```

Comment:
Don't use an empty array for the URL and the PATH; better to create an `anyURL()` and `anyPath()` function with a random string.

Reply:
done